src/Rules/CnesXMLIdentification.php

```diff
@@ -10,9 +10,10 @@ class CnesXMLIdentification implements Rule
 {
 	private $expected_ibge_code;
 
-	public function __construct($expected_ibge_code)
+	public function __construct($expected_ibge_code, $version_xsd)
```

Comment:
No `src/Rules/ZipHasValidCnesXML.php` o valor do version_xsd é opcional (padrão `null`) e aqui não é opcional, isso vai quebrar a compatibilidade com quem está utilizando a versão anterior pq a regra vai lançar exceção quando o  `src/Rules/ZipHasValidCnesXML.php` for chamado sem passar uma versão. 

Para manter a compatibilidade vejo duas alternativas:
1. Adicionar a versão `2.1` como padrão no argumento do `ZipHasValidCnesXML`
2. Deixar o `version_xsd` como opcional no `CnesXMLIdentification.php` e só fazer a verificação dele se for diferente de `null`.